golem/ethereum/incomeskeeper.py

```diff
@@ -208,6 +208,7 @@ def update_overdue_incomes() -> None:
             return
 
         for income in incomes:
+            logger.debug("Marking payment as overdue: sender: %s, amount: %s ")
```

Comment:
```suggestion
            logger.debug("Marking payment as overdue. sender=%s, amount=%s ")
```
would format data as kwargs, according to logging guide

also no values are passed for the formatters.